tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -179,6 +180,49 @@ def test_fsx_lustre(
     )
 
 
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+def test_fsx_lustre_dra(
```

Comment:
I considered adding it to test_fsx_luster but since that uses config files with import/export path it got a bit complicated because I would have to unmount the old file system and then mount a new one with dras